.gitlab-ci.yml

```diff
@@ -8,7 +8,7 @@ stages:
   - deploy-special
 
 variables:
-  KUBESPRAY_VERSION: v2.18.1
+  KUBESPRAY_VERSION: v2.19.1
```

Comment:
This change causes a CI error.
How about just keeping here as is and updating README.md part only in this PR?

Reply:
😊 Done.